egs/gale_arabic/s5b/local/gale_prep_grapheme_dict.sh

```diff
@@ -16,7 +16,7 @@ bzcat ar-ar_grapheme_lexicon_2016-02-09.bz2  | sed '1,3d' | awk '{print $1}'  >
 bzcat ar-ar_lexicon_2014-03-17.txt.bz2 | sed '1,3d' | awk '{print $1}' >>  tmp$$
 # (2) Now we add all the words appeared in the training data
 cat data/local/train/text | cut -d ' ' -f 2- | tr -s " " "\n" | sort -u >> tmp$$
-grep -v [0-9] tmp$$ |  sort -u > tmp1.$$
+grep -v [0-9] tmp$$ |  sed -e 's:[FNKaui\~o\`]::g' -e 's:{:}:g' | sort -u > tmp1.$$ # remove vowels and  rare alef wasla
```

Comment:
Got some errors as below: 
328 Checking data/local/dict/lexiconp.txt                                           
329 --> reading data/local/dict/lexiconp.txt                                        
330 --> ERROR: phone "a" is not in {, non}silence.txt (line 21515)                  
331 --> ERROR: phone "a" is not in {, non}silence.txt (line 33745)                  
332 --> ERROR: phone "a" is not in {, non}silence.txt (line 455372)                 
333 --> ERROR: phone "a" is not in {, non}silence.txt (line 455373)                 
334 --> ERROR: phone "a" is not in {, non}silence.txt (line 455374)                 
335 --> ERROR: phone "a" is not in {, non}silence.txt (line 455375)                 
336 --> ERROR: phone "a" is not in {, non}silence.txt (line 455376)                 
337 --> ERROR: phone "a" is not in {, non}silence.txt (line 630905)                 
338 --> ERROR: phone "{" is not in {, non}silence.txt (line 955030)                 
339 --> ERROR: phone "{" is not in {, non}silence.txt (line 955031)                 
340 --> ERROR: phone "{" is not in {, non}silence.txt (line 955032)                 
341                                                                                 
342 Checking lexicon pair data/local/dict/lexicon.txt and data/local/dict/lexiconp.txt
343 --> ERROR: data/local/dict/lexicon.txt and data/local/dict/lexiconp.txt mismatch at line 21515. sorting?
344 Checking data/local/dict/extra_questions.txt ...                                
345 --> data/local/dict/extra_questions.txt is empty (this is OK)                   
346 --> ERROR validating dictionary directory data/local/dict (see detailed error messages above)
347                                                                                 
348 *Error validating directory data/local/dict*